examples/complex/.babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["es2015", "stage-2", "stage-3"],
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "8.9"
```

Comment:
We should review this at some point and see if we shouldn't be targetting browsers as well.